src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -247,24 +247,28 @@ def _list_devices(self, host=None, format='plain', refresh=False):
         else:
             out = []
 
+            table = PrettyTable(
+                ['HOST', 'PATH', 'TYPE', 'SIZE', 'DEVICE', 'AVAIL',
+                 'REJECT REASONS'],
+                border=False)
+            table.align = 'l'
+            table._align['SIZE'] = 'r'
+            table.left_padding_width = 0
+            table.right_padding_width = 1
             for host in completion.result: # type: orchestrator.InventoryNode
-                out.append('Host {}:'.format(host.name))
-                table = PrettyTable(
-                    ['Path', 'Type', 'Size', 'Available', 'Ceph Device ID', 'Reject Reasons'],
-                    border=False)
-                table._align['Path'] = 'l'
                 for d in host.devices.devices:  # type: Device
                     table.add_row(
                         (
+                            host.name,
                             d.path,
                             d.human_readable_type,
                             format_bytes(d.sys_api.get('size', 0), 5, colored=False),
```

Comment:
```suggestion
                             format_bytes(d.sys_api.get('size', 0), 5),
```

`colored=False` is the default now.